lib/symengine/complex.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class Complex
+    def to_c
+        to_s.tr('I', 'i').tr('*','').tr(' ','').to_c
```

Comment:
This is pretty minor, but you can use one `tr('* ', '')` instead of two, when replacing chars with nothing.
Oh! Last moment thought: just `String#delete` would be even better :)